templates/compose/nodebb.yaml

```diff
@@ -0,0 +1,60 @@
+# ignore: false
+# documentation: https://docs.nodebb.org/
+# slogan: A next-generation discussion platform.
+# tags: communication, forums, discussion
+# logo: svgs/nodebb.svg
+# port: 4567
+
+services:
+  nodebb:
+    image: ghcr.io/nodebb/nodebb:latest
+    restart: unless-stopped
+    environment:
+      - SERVICE_FQDN_nodebb_4567
+    volumes:
+      - nodebb-build:/usr/src/app/build
+      - nodebb-uploads:/usr/src/app/public/uploads
+      - nodebb-config:/opt/config
+      - type: bind
+        source: ./.docker/config/setup.json
+        target: /usr/src/app/setup.json
+        isDirectory: false
+        content: |
+          {
+              "defaults": {
+                  "postgres": {
+                      "host": "postgres",
+                      "port": 5432,
+                      "database": "nodebb",
+                      "username": "nodebb",
+                      "password": "nodebb"
+                  }
+              }
+          }
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:4567"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 30s
+
+  postgres:
+    image: postgres:17.2-alpine
+    restart: unless-stopped
+    environment:
```

Comment:
Please use the Coolify magic variables for the password, username, etc. https://coolify.io/docs/knowledge-base/docker/compose#coolifys-magic-environment-variables

Reply:
Added the magic variables where applicable and made a new entrypoint script to autofill the json file with the proper variables